tests/src/test/scala/cats/tests/ValidatedTests.scala

```diff
@@ -2,13 +2,42 @@ package cats
 package tests
 
 import cats.data.Validated
+import cats.data.Validated.{Valid, Invalid}
 import cats.std.string._
 import cats.laws.discipline.{ApplicativeTests, SerializableTests}
 import cats.laws.discipline.arbitrary._
 import org.scalacheck.Arbitrary
+import org.scalacheck.Prop._
 
 class ValidatedTests extends CatsSuite {
-
   checkAll("Validated[String, Int]", ApplicativeTests[Validated[String,?]].applicative[Int, Int, Int])
   checkAll("Applicative[Validated[String,?]]", SerializableTests.serializable(Applicative[Validated[String,?]]))
+
+  test("ap2 combines failures in order") {
+    val plus = (_: Int) + (_: Int)
+    assert(Validated.validatedInstances[String].ap2(Invalid("1"), Invalid("2"))(Valid(plus)) == Invalid("12"))
+  }
+
+  test("fromTryCatch catches matching exceptions") {
+    assert(Validated.fromTryCatch[NumberFormatException]{ "foo".toInt }.isInstanceOf[Invalid[NumberFormatException]])
+  }
+
+  test("fromTryCatch lets non-matching exceptions escape") {
+    val _ = intercept[NumberFormatException] {
```

Comment:
without something extra compile fails with "discarded non-Unit value".

I think val _ is the clearest way to express that the value is discarded intentionally but I'm happy to use some other pattern if there's a convention.